id = 1333
title = "vhost-user-test qos-test fails on s390x host"
state = "closed"
created_at = "2022-11-22T18:09:34.151Z"
closed_at = "2022-12-04T23:45:56.250Z"
labels = ["host: s390", "target: arm"]
url = "https://gitlab.com/qemu-project/qemu/-/issues/1333"
host-os = "Ubuntu 20.04"
host-arch = "s390x"
qemu-version = "commit 16a550bdc0e49fcda0e6a6c55d648700ad33c8a4"
guest-os = "n/a"
guest-arch = "n/a"
description = """The qos-test is now definitely failing in the ubuntu-20.04-s390x-all CI job. See https://gitlab.com/qemu-project/qemu/-/jobs/3363173491 , then click on "Complete Raw" to see the full log. Quoting:

```
ERROR:../tests/qtest/vhost-user-test.c:248:wait_for_fds: assertion failed: (s->fds_num)
**
ERROR:../tests/qtest/qos-test.c:191:subprocess_run_one_test: child process (/arm/virt/virtio-mmio/virtio-bus/vhost-user-gpio-device/vhost-user-gpio/vhost-user-gpio-tests/read-guest-mem/memfile/subprocess [274051]) failed unexpectedly

(test program exited with status code -6)
```"""
reproduce = "n/a"
additional = "n/a"
